src/core/GameKeyHandler.ttslua

```diff
@@ -98,14 +99,18 @@ function takeClueFromLocation(playerColor, hoveredObject)
     return
   end
 
+  local clickableClues = optionPanelApi.getOptions()["useClueClickers"]
   local playerName = Player[playerColor].steam_name
-  local matColor = playmatApi.getMatColor(playerColor) 
-  local pos = playmatApi.transformLocalPosition({x = -1.12, y = 0.05, z = 0.7}, matColor)
-  local rot = playmatApi.returnRotation(matColor)
-
-  if cardName == "" or cardName == nil then
-    cardName = "nameless card"
```

Comment:
Why was this removed?

Reply:
Because it conflicts with the if statement a few lines down (LOC 123-127).  Of the two options, I thought the later one made more sense, since stating you received a clue from a 'nameless card' is not very useful, so you may as well just say nothing about the card at all.  I can switch it back if preferred though, but I still think we should have only one or the other for code clarity.